### PR TITLE
0.22.0 changelog

### PR DESCRIPTION
**Pull request by fboucquez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
...
+## [0.22.0] - 3-Dec-2020
+
+**Milestone**: Catapult-server finality(0.10.0.4)
+ Package  | Version  | Link
+---|---|---
+SDK Core| v0.22.0 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
+Catbuffer | v0.0.24 | [catbuffer-typescript](https://www.npmjs.com/package/catbuffer-typescript)
+Client Library | v0.10.4  | [symbol-openapi-typescript-fetch-client](https://www.npmjs.com/package/symbol-openapi-typescript-fetch-client)
+
+- **[BREAKING CHANGE]** `Deadline.create` requires the configurable `epochAdjustment` from the network properties.
```
Comment (rg911) [inline CHANGELOG.md:16]: would be better to mention epochAdjustment can be retrieved from `network/properties` endpoint or using the repositoryFactory.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
...
+- **[BREAKING CHANGE]** `NetworkCurrency` subclasses replaced with `Currency` objects.
+- **[BREAKING CHANGE]** `SecreatLockRepository.getSecretLock` has been removed. You can now search by secret by using the search criteria.
```
Comment (rg911) [inline CHANGELOG.md]: Also breaking change for new block types

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +20,15 @@ Client Library | v0.10.1  | [symbol-openapi-typescript-fetch-client](https://www
 - Added `transferMosaicId`, `fromTransferAmount`, `toTransferAmount` to transaction searches.
 - Added `CurrencyService` to allow loading Network and custom `Currency` objects from the rest service.
 - Added `UnlockedAccount` endpoint in `NodeRepository` for checking delegated harvesting status.
+- Added `StateProofService` to verify the different states.
+- Added `serialize()` to state objects like `AccountInfo`, `MosaicInfo`, etc.
```
Comment (rg911) [inline CHANGELOG.md:24]: in 
AccountInfor
MosaicInfo
NamespaceInfo
MultisigInfo
AccountRetrictionInfo
MosaicRestrictionInfo
MetadataEntry
SecretLockInfo
HashLockInfo

to generate state proof hash

File: CHANGELOG.md [hunk 4]
```diff
@@ -20,6 +20,15 @@ Client Library | v0.10.1  | [symbol-openapi-typescript-fetch-client](https://www
...
+- Added `version` field to state objects.
+- Added `/merkle` endpoints to the repositories of the different states.
+- Added `stremer()` to repositories to simplify `PaginationStreamer` objects creation.
+- Improved `search` endpoints allowing "empty" criteria in order to paginate over all the objects.
+- `Listener` now accepts address aliases as `UnresolvedAddress` objects.
+- Added V1 and V2 Voting Key transaction support.
+- Updated `FinalizationProof` objects to the latest server changes.
```
Comment (rg911) [inline CHANGELOG.md:31]: Updated `FinalizationProof` object added `SignatureSchema` for server tree `testnet/v3`

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,14 +4,14 @@ All notable changes to this project will be documented in this file.
...
 ---|---|---
-SDK Core| v0.21.1 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
-Catbuffer | v0.0.22 | [catbuffer-typescript](https://www.npmjs.com/package/catbuffer-typescript)
-Client Library | v0.10.1  | [symbol-openapi-typescript-fetch-client](https://www.npmjs.com/package/symbol-openapi-typescript-fetch-client)
+SDK Core| v0.22.0 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
+Catbuffer | v0.0.24 | [catbuffer-typescript](https://www.npmjs.com/package/catbuffer-typescript)
+Client Library | v0.10.4  | [symbol-openapi-typescript-fetch-client](https://www.npmjs.com/package/symbol-openapi-typescript-fetch-client)
 
 - **[BREAKING CHANGE]** `Deadline.create` requires the configurable `epochAdjustment` from the network properties.
 - **[BREAKING CHANGE]** `NetworkCurrency` subclasses replaced with `Currency` objects.
```
Comment (rg911) [inline CHANGELOG.md:17]: Also BreakingChange for new block types

**Review by rg911 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (fboucquez) [reply CHANGELOG.md]: Is new block types breaking change? or just more info on the existing?

**Review by rg911 — APPROVED** [review]

**Pull request merged**